tests/subsys/a_mod/prj.conf

```diff
@@ -0,0 +1,15 @@
+CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
+CONFIG_ZTEST_STACK_SIZE=16384
+CONFIG_IRQ_OFFLOAD=y
+CONFIG_AUDIO_MODULE=y
+
+# Added large stack sizes. Can be optimized.
+CONFIG_MAIN_STACK_SIZE=32768
+
+CONFIG_STACK_SENTINEL=y
+
+CONFIG_THREAD_ANALYZER=n
```

Comment:
I think this is default "no" so it can be removed.

Reply:
Also, if this is "no" then the next three lines can be removed as well.

Reply:
Sure, I was keeping them there while testing.